obspy/core/inventory/response.py

```diff
@@ -1663,6 +1663,42 @@ def get_sacpz(self):
         paz = self.get_paz()
         return paz_to_sacpz_string(paz, self.instrument_sensitivity)
 
+    @staticmethod
+    def from_paz(zeros, poles, stage_gain,
+                 stage_gain_frequency=1.0, input_units='VOLTS',
+                 output_units='M/S', normalization_frequency=None,
+                 pz_transfer_function_type='LAPLACE (RADIANS/SECOND)',
+                 normalization_factor=1.0):
+        """
+        Takes in lists of complex poles and zeros and returns a Response with
+        those values defining its only stage.
+        :type zeros: list of complex
+        :param zeros: All zeros of the response to be defined.
+        :type poles: list of complex
+        :param poles: All poles of the response to be defined.
+        :type stage_gain: double
+        :param stage_gain: The gain value of the response [sensitivity]
+
+        Most of the optional parameters defined here are from
+        :class:`~obspy.core.inventory.response.PolesZerosResponseStage`
+        Note that if no value is specified for the normalization_frequency
+        parameter, the value assigned to stage_gain_frequency will be used.
+        :returns: new Response instance with given P-Z values
+        """
+        if normalization_frequency is None:
```

Comment:
This is very unlikely to happen it defaults to 1.0 so people would have to explicitly set it to `None`. In that case I guess they are just more likely to directly set it to whatever value they want. I would remove this "magic" behavior and just assume people are very explicit in what they want.